src/Compilers/CSharp/Portable/Binder/Binder_Invocation.cs

```diff
@@ -1688,7 +1688,8 @@ private BoundFunctionPointerInvocation BindFunctionPointerInvocation(SyntaxNode
                     methods,
                     typeContainingConstructor: null,
                     delegateTypeBeingInvoked: null,
-                    returnRefKind: funcPtr.Signature.RefKind);
+                    returnRefKind: funcPtr.Signature.RefKind,
+                    delegateOrFunctionPointerType: funcPtr);
```

Comment:
It was very strange to me to see an existing funcPtr and not passing it here. However, I'm not sure how this would affect the behavior. We may need a test that have different behavior before and after this change.